CHANGELOG.md

```diff
@@ -54,6 +54,8 @@
   lookup errors instead of silently ignoring them.
 - [#2999](https://github.com/ChainSafe/forest/issues/2999): Restored `--tipset`
   flag to `forest-cli snapshot export` to allow export at a specific tipset.
+- [#3283](https://github.com/ChainSafe/forest/pull/3283): All generated car
```

Comment:
It's a breaking change, right?

Reply:
In other words, should any services (e.g., upload snapshot) be updated to this logic?

Reply:
Are we also good with extensions and many regular expressions to match files?

Reply:
Oh,yeah, we'll need to update some regexes.